### ISSUE TITLE
CSV format not correct for newer scenarios

### ISSUE THREAD

**Issue by jriall** [fat-dash/fat-dash#13]
I am unable to import stats for a number of scenarios. Currently trying to import in the UI, I get `cannot import 421 scenarios`. In the browser console I get 421 of these errors:

```
"app:user:error FuglaaXYZ Voltaic Easy - Challenge - 2020.12.29-11.11.56 Stats.csv: CSV format not correct. Expected key "undefined". Actual is "Challenge Start:"."
```

This seems to be happening only for newer scenarios I've been following (new Voltaic benchmarks). Perhaps a Kovaaks update has changed the CSV format of newly created scenarios?

**Comment by vrnvorona** [comment]
Same issue

**Comment by blahokay1** [comment]
I'm having this issue for all scenarios since the newest steam update.

**Comment by jriall** [comment]
Yes, same here, in the original issue I posted I thought it was for newer scenarios, but that was only because I was currently only playing new scenarios. This is broken for all scenarios it seems. I'm guessing still that it's simply a CSV format change. I might be able to dig into this and create a PR later on potentially.

**Comment by jriall** [comment]
Oh wait, I can't do that as this repo only has the compiled code uploaded and not the src folder...

Does anyone know where the src for this project is?
